nixos/modules/services/networking/netboot.nix

```diff
@@ -0,0 +1,29 @@
+{ config, pkgs, lib }:
+
+let
+  configuration = with lib; {
+    imports = [
+        (pkgs.path + "/nixos/modules/installer/netboot/netboot-minimal.nix")
+    ];
+    ## Some useful options for setting up a new system
+    services.mingetty.autologinUser = mkForce "root";
```

Comment:
This might be a reasonable default, but some users might want to disable it. Is there a way to apply it without `mkForce`?